WindSpigot-Server/src/main/java/ga/windpvp/windspigot/config/WindSpigotConfig.java

```diff
@@ -81,9 +77,44 @@ public static void init(File configFile) {
 		}
 		
 	}
-	
-	// Not private as the config is read by calling all private methods with 0 params
-	static void makeReadable() {
+
+	private static final Map<String, String> oldToNewConfigNames = new HashMap<String, String>() {{
+		put("settings.fixed-pools.size", "settings.async.tnt.pool-size");
```

Comment:
```suggestion
		put("settings.fixed-pools.size", "settings.async.explosions.enable");
```
This is the new path

Reply:
I am incorrect, however it's not that but instead `settings.async.explosions.threads`
```suggestion
		put("settings.fixed-pools.size", "settings.async.explosions.threads");
```

Reply:
Oh XD wrong setting